src/features/search/components/SearchResults.tsx

```diff
@@ -211,9 +211,13 @@ const FilterContainer = styled.View(({ theme }) => ({
   zIndex: theme.zIndex.floatingButton,
 }))
 
-const ScrollToTopContainer = styled(FilterContainer)({
+const ScrollToTopContainer = styled.View(({ theme }) => ({
+  alignSelf: 'center',
+  position: 'absolute',
   right: getSpacing(7),
-})
+  bottom: theme.tabBar.height + getSpacing(6),
+  zIndex: theme.zIndex.floatingButton,
+}))
```

Comment:
J'ai du mal à comprendre ce que ces changements impliquent, même si d'après le message de commit ce ne sont que des changements mineurs :)

Reply:
`ScrollToTopContainer` reprenait des propriétés CSS de `FilterContainer`

j'imagine pour éviter de la duplication ?

ça créait une dépendance entre ces 2 composants, qui n'avait pas de sens : ces composants ne sont pas liés

---

> for each desired change, make the change easy (warning: this may be hard), then make the easy change
> [@KentBeck](https://twitter.com/KentBeck/status/250733358307500032)

je fais régulièrement du refactoring en amont, avant de faire la feature que je veux

dans un futur proche, je vais supprimer `FilterContainer` (le commit est fait, mais le reste de la branche n'est pas prete)

j'ai donc cassé la dépendance qui n'avait pas lieu d'etre